docs/modules/ic-identity-guide/pages/auth-how-to.adoc

```diff
@@ -6,21 +6,23 @@
 :company-id: DFINITY
 :sdk-short-name: DFINITY Canister SDK
 
-Internet Identity enables you to authenticate securely and anonymously when you access applications that use it as an authentication method. A different pseudonym is created for each application you log in to, and you will be able to use all of your registered devices or authentication methods to log in to the same identity. This is all accomplished using encrypted smart contracts that run on the {IC} blockchain.
+The Internet Identity enables you to authenticate securely and anonymously when you access applications that use it as an authentication method. A different identity is created for each application you log in to, and you will be able to use all of your registered devices or authentication methods to log in to the same account. This is all accomplished because Internet Identity is built to run on the {IC} blockchain, ensuring cryptographic security, and leverages the browser standard, Web Authentication (WebAuthn). 
 
-Unlike most authentication methods, your Internet Identity does not require you to set and manage passwords or provide any personal identifying information to applications or to Internet Identity. Instead, you create authentication profiles using an ID number and the authentication methods you choose such as facial recognition from a smartphone, your computer unlock password, or a security key. 
+Unlike most authentication methods, your Internet Identity does not require you to set and manage passwords or provide any personal identifying information to applications or to Internet Identity. Instead, you register a device by using any of the device's own authentication methods. If the device supports WebAuthn, and accepts your authentication method, a public key is created and is represented by an ID number (you User Number). At the same time, the security chip in your device generates a  private key, which creates a signature associated with your ID number. The private key is secret and never leaves your device. No one, not even you, can see the private key. The unique signature from your private key is what Internet Identity uses to prove that you are in possession of the correct private key for your ID number. 
```

Comment:
Dom explicitly wanted to drop the term "user number".

Reply:
I know, but until the UI is changed, I have to refer to it. 
And "pseudonym" literally means "fake name" - I can't use it in good conscience.

Reply:
I am just looking at this thread , and see that "user" doesn't sound good, and pseudonym doesn't sound good , what about calling them I iidentities .  Like a double identity.  The people can be ii's, and one person can have another identity.  Plural of I.  
Perhaps people will have iiiiiiiiidenties....

Reply:
My identity uses an iPhone . The internet computer identity app could be called "ii" ... there's a lot that could be done with the term internet identity and the shortened version of it "ii"